BUILD

```diff
@@ -105,6 +107,14 @@ deploy_pip(
 )
 
 
+deploy_github(
+    name = "deploy-github",
+    targets = [],
+    release_description = "//:RELEASE_TEMPLATE.md",
+    deployment_properties = "//:deployment.properties",
```

Comment:
we don't need to pass a whole file (`deployment.properties`) just to pass the address to the GitHub repo here. I think a string argument containing the repo address is enough and simpler.

Reply:
it turns out we do need it